pkg/plugins/resources/gitlab/release/main.go

```diff
@@ -53,7 +53,7 @@ type Gitlab struct {
 	versionFilter version.Filter
 }
 
-// New returns a new valid Github object.
+// New returns a new valid GitHub object.
```

Comment:
```suggestion
// New returns a new valid GitLab object.
```

That one sounds like a copy and paste when the `Gitlab` resource was created from `GtiHub`. Using the "brand" case is ok, even if it is a technical reference